tests/run_tests/driver_airborne_fem_1d_test.py

```diff
@@ -44,18 +54,17 @@ def test_fem_fwr_1d_run(
     cell_size=(20.0, 20.0, 20.0),
 ):
     # Run the forward
+    opts = SyntheticDataInversionOptions(
+        survey=SurveyOptions(
+            n_stations=n_grid_points, n_lines=n_grid_points, drape=10.0
+        ),
+        mesh=MeshOptions(
+            cell_size=cell_size, refinement=refinement, padding_distance=400.0
+        ),
+        model=ModelOptions(background=1e-4, anomaly=0.1),
+    )
     geoh5, _, model, survey, topography = setup_inversion_workspace(
```

Comment:
```suggestion
    geoh5, mesh, model, survey, topography = setup_inversion_workspace(
```
Can you change them all to use the `mesh` directly instead of `model.parent`?

Reply:
refactor will adress this issue